android/festago/app/src/main/java/com/festago/festago/FestagoApplication.kt

```diff
@@ -10,9 +14,21 @@ class FestagoApplication : Application() {
     override fun onCreate() {
         super.onCreate()
         initKakaoSdk()
+        initNotificationChannel()
     }
 
     private fun initKakaoSdk() {
         KakaoSdk.init(this, BuildConfig.KAKAO_NATIVE_APP_KEY)
     }
+
+    private fun initNotificationChannel() {
+        val notificationManager =
+            getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        val channel = NotificationChannel(
+            FcmMessageType.ENTRY_ALERT.channelId,
+            getString(R.string.entry_alert_channel_name),
+            NotificationManager.IMPORTANCE_DEFAULT
+        )
```

Comment:
```suggestion
        val channel = NotificationChannel(
            FcmMessageType.ENTRY_ALERT.channelId,
            getString(R.string.entry_alert_channel_name),
            NotificationManager.IMPORTANCE_DEFAULT
        )
        val notificationManager =
            getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
```
사용하기 직전에 꺼내오는 것이 어떨까요??

Reply:
반영완료!